main.nf

```diff
@@ -528,18 +594,60 @@ if (!params.skip_alignment) {
         samtools flagstat ${sample}.sorted.bam > ${sample}.sorted.bam.flagstat
         samtools idxstats ${sample}.sorted.bam > ${sample}.sorted.bam.idxstats
         samtools stats ${sample}.sorted.bam > ${sample}.sorted.bam.stats
-        samtools --version &> samtools.version
         """
     }
-} else {
-    ch_graphmap_version = Channel.empty()
-    ch_minimap2_version = Channel.empty()
-    ch_samtools_version = Channel.empty()
-    ch_sortbam_stats_mqc = Channel.empty()
+
+    /*
+     * STEP 9 - Convert BAM to BEDGraph
+     */
+    process BAMToBedGraph {
+        tag "$sample"
+        label 'process_medium'
+        publishDir path: "${params.outdir}/${params.aligner}/bigwig/", mode: 'copy',
+            saveAs: { filename ->
+                          if (filename.endsWith(".bedGraph")) filename
+                    }
+
+        input:
+        set file(fasta), file(sizes), val(sample), file(bam) from ch_sortbam_bam
+
+        output:
+        set file(fasta), file(sizes), val(sample), file("*.bedGraph") into ch_bedgraph
+        file "*.version" into ch_bedtools_version
+
+        script:
+        """
+        genomeCoverageBed -ibam ${bam[0]} -bg | sort -k1,1 -k2,2n >  ${sample}.bedGraph
+        bedtools --version > bedtools.version
```

Comment:
With this section, whether to normalise data or not normalise is something that we might need to make an option - it can be much more informative when looking at individual runs to get the absolute data. 
Also it won't affect data with standard chromosomes, but for data with non-standard chromosomes a `bedSort` between the `genomeCoverageBed` and the later `bedGraphToBigWig` will ensure it doesn't break. 
At least that's what I was sorta thinking here: https://github.com/lwratten/nanoseq/blob/67bd4ec20179824292fb26a707dc07a39031695f/main.nf#L538-L540

Reply:
Thats what the pipe to `sort -k1,1 -k2,2n` is doing?

Reply:
Will have to think about the normalisation abit more because the easiest solution is to use the number of mapped reads from the `flagstat` files and then scale based on that:
https://github.com/nf-core/chipseq/blob/21be3149542cdc84431e12d1e092359058aed32a/main.nf#L924-L926

However, for long reads you will get supplementary alignments etc which tend to complicate how the total mapped reads are counted, and so we would need to get a unique count of the total mapped reads somehow. Lets leave this for another PR 👍